src/base/server_type.cpp

```diff
@@ -104,6 +107,37 @@ bool is_tran_server_with_remote_storage ()
   return false;
 }
 
+void set_server_type_from_arg (int argc, char **argv)
+{
+  if (argc < 4)
+    {
+      return;
+    }
+
+  int arg = 0;
+  //check for server type argument
+  for (size_t i = 2; i < argc; i++)
+    {
+      if (arg)
+	{
+	  if (!strcmp (argv[i], "transaction"))
+	    {
+	      g_server_type = SERVER_TYPE_TRANSACTION;
+	      return;
+	    }
+	  if (!strcmp (argv[i], "page"))
```

Comment:
```suggestion
	  if (strcmp (argv[i], "page") == 0)
```
Does lowercase comparison conform to cubrid conventions regarding argument casing? I checked and `cubrid sErViCe StarT` works (`cubrid server start SCALDB` does not).